src/comm/USBBoardInfo.json

```diff
@@ -21,7 +21,7 @@
         { "vendorID": 1155, "productID": 41775,     "boardClass": "Pixhawk",    "name": "PX4 FMU ModalAI FCv1" },
         { "vendorID":12642, "productID": 75,        "boardClass": "Pixhawk",    "name": "PX4 DurandalV1" },
         { "vendorID": 4104, "productID": 1,         "boardClass": "Pixhawk",    "name": "PX4 FMU UVify Core" },
-        { "vendorID": 12643, "productID": 76,       "boardClass": "Pixhawk",    "name": "PX4 CUAV X7(PRO)" },
+        { "vendorID": 12643, "productID": 76,       "boardClass": "Pixhawk",    "name": "CUAV Flight Controller" },
```

Comment:
@czhj Just double-checking: The VID/PIDs for all your boards are the same?

Reply:
yes, all boards use the same usb id, because it is very expensive.